Makefile

```diff
@@ -1,6 +1,7 @@
-.PHONY: help install test tests update dist publish
+.PHONY: help install test tests update dist publish promote
 SHELL=/bin/bash
 ECR_REGISTRY=672626379771.dkr.ecr.us-east-1.amazonaws.com
+DATETIME:=$(shell date -u +%Y%m%dT%H%M%SZ)
```

Comment:
When `make` is run this just gets populated once at runtime? I think so, but just checking.

Reply:
Yeah: https://www.gnu.org/software/make/manual/make.html#Flavors